tests/services/test_crash_reports.py

```diff
@@ -0,0 +1,61 @@
+import glob
+import shutil
+import time
+
+import pytest
+
+from pymobiledevice3.services.crash_reports import CrashReportsManager
+
+
+@pytest.fixture(scope='function')
+def crash_manager(lockdown):
+    return CrashReportsManager(lockdown)
+
+
+def test_ls_default(crash_manager):
+    crash_manager.afc.makedirs('temp')
+    assert '/temp' in crash_manager.ls()
+
+
+def test_ls_path(crash_manager):
+    crash_manager.afc.makedirs('temp/temp')
+    assert '/temp/temp' in crash_manager.ls(path='/temp')
+
+
+@pytest.mark.parametrize('depth', [2, 3, 4])
+def test_ls_depth(crash_manager, depth):
+    path = ''
+    path_list = []
+    for i in range(depth):
+        path += f'/temp'
+        path_list.append(path)
+    crash_manager.afc.makedirs(path)
+    crash_list = crash_manager.ls(depth=depth)
+    for item in path_list:
```

Comment:
why not `assert path_list == crash_list`?

Reply:
because it can be with more crash reports files or dirs inside crash_list...

so we want just to ensure the dirs we created